Dockerfile

```diff
@@ -134,7 +134,9 @@ WORKDIR $APP_HOME
 
 COPY --from=clam_builder "/clamav" "/"
 
-RUN ln -s /usr/bin/clam* /usr/local/bin
+RUN ln -s /usr/bin/clam* /usr/local/bin && \
+    ln -s /usr/bin/freshclam /usr/local/bin && \
+    chown 0755 /usr/local/bin/clam* /usr/local/bin/freshclam
```

Comment:
Did you mean chmod? Not sure what's intended there though because the mode of the binaries should already be fine (like 0755 or something) and the mode of the links themselves is also gonna be fine by default. If the binaries being symlinked to are 0700 or something then `chmod 755` on them would make sense (but preferably not via the links, because that'd be a little confusing to someone reading the script).

Reply:
Whoops yes, wrong command name. 🤦